taxi/tests/test_views.py

```diff
@@ -0,0 +1,111 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+
+MANUFACTURER_URL = reverse("taxi:manufacturer-list")
+DRIVER_URL = reverse("taxi:driver-list")
+CAR_URL = reverse("taxi:car-list")
+
+
+class PublicTest(TestCase):
+    def test_login_required_for_manufacturer_list_page(self):
+        response = self.client.get(MANUFACTURER_URL)
+        self.assertNotEqual(response.status_code, 200)
```

Comment:
In tests, it's better to assert for exact expected status codes. In this case, you should expect a 302 redirect status code (since the user is not logged in and should be redirected to the login page). Instead of using `assertNotEqual`, use `assertEqual` to compare the response status code with 302. This will make the test more accurate.